### DISCUSSION TITLE
MacOS Sequoia (With AD users) - brew Installation failed

### DISCUSSION THREAD

**Discussion by snowcube-ui** [Homebrew/discussions#5641] [Getting started]
### Output of `brew config`

```shell
can't provide - since brew is not installed
```

### Output of `brew doctor`

```shell
can't provide - since brew is not installed
```

### Description of issue

Hey Team, 
strange behavior getting while installing HomeBrew on MacOS enrolled into Intune and used with mobile account from Directory Services:

here is output modified about user names (some are user from AD and some are admins account from AD) 

user@NB28063086 ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R ADadmin:admin /opt/homebrew
==> /usr/bin/sudo /usr/sbin/chown -R ADadmin /Users/user/Library/Caches/Homebrew
touch: /Users/user/Library/Caches/Homebrew/.cleaned: Permission denied
Failed during: /usr/bin/touch /Users/user/Library/Caches/Homebrew/.cleaned

when I list file .cleaned I got this

user@NB28063086 ~ % ll /Users/user/Library/Caches/Homebrew/.cleaned
-rw-r--r--  1 ADadmin  DOMAIN\Domain Users  0 28 zář 13:20 /Users/user/Library/Caches/Homebrew/.cleaned

after several attempts to fix rights ... I got this:

user@NB28063086 ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew
==> The following existing directories will be made writable by user only:
/opt/homebrew/share/zsh
/opt/homebrew/share/zsh/site-functions
==> The following existing directories will have their owner set to ADadmin:
/opt/homebrew/share/zsh
/opt/homebrew/share/zsh/site-functions
==> The following existing directories will have their group set to admin:
/opt/homebrew/share/zsh
/opt/homebrew/share/zsh/site-functions

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /bin/chmod u+rwx /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions
==> /usr/bin/sudo /bin/chmod go-w /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions
==> /usr/bin/sudo /usr/sbin/chown ADadmin /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions
==> /usr/bin/sudo /usr/bin/chgrp admin /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions
==> /usr/bin/sudo /usr/sbin/chown -R ADadmin:admin /opt/homebrew
==> /usr/bin/sudo /usr/sbin/chown -R ADadmin /Users/user/Library/Caches/Homebrew
==> Downloading and installing Homebrew...
fatal: not in a git directory
Failed during: /usr/bin/git config remote.origin.url https://github.com/Homebrew/brew

can I specify somewhere directly User (I would provide AD user)  a local Administrator (root) not ADadmin ... ???

thanks.

**Comment by SMillerDev** [comment]
I'd recommend using the pkg installer in an enterprise context. Or workbrew I guess
